src/spark_rapids_ml/common/cuml_context.py

```diff
@@ -50,6 +50,8 @@ def __init__(
         """
         self.enable = enable
         self._handle: Optional[Handle] = None
+        self._loop: Optional[AbstractEventLoop] = None
```

Comment:
RandomForest doesn't enable NCCL, but the _loop has been referred in the common core

Reply:
Did the unit tests not catch this?

Reply:
Unit tests currently fail